public/index.html

```diff
@@ -24,6 +37,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script
+      src="https://kit.fontawesome.com/cc56ff6206.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
Why fontawesome? I've added `MUI Icons` in the package last night. Use that if u need icons.

Reply:
ok vaia